src/main/index.ts

```diff
@@ -25,6 +35,9 @@ const createWindow = (): void => {
 // Some APIs can only be used after this event occurs.
 app.on('ready', createWindow);
 
+// TODO: delete this exemplary code
+app.on('browser-window-blur', () => store.dispatch(exampleAliasedAction('text')))
```

Comment:
A simple action dispatched from the `main` process.  
Notice that it is aliased -> is only being executed in the `main` process, and the result of which is being broadcast to the `renderer` processes.